torch/nn/intrinsic/qat/modules/linear_relu.py

```diff
@@ -36,3 +37,12 @@ def forward(self, input):
     @classmethod
     def from_float(cls, mod):
         return super(LinearReLU, cls).from_float(mod)
+
+    def to_float(self):
+        linear = torch.nn.Linear(self.in_features, self.out_features)
```

Comment:
should we respect the device and is_training settings when creating the new module?